specification/network/resource-manager/Microsoft.Network/stable/2021-03-01/virtualWan.json

```diff
@@ -5448,6 +5448,240 @@
           "final-state-via": "location"
         }
       }
+    },
```

Comment:
virtualHubs 2021-03-01 has already been added to the Microsoft.Network manifest (https://armmanifest.visualstudio.com/One/_git/Manifest?path=%2FMICROSOFT.NETWORK%2FPROD%2FMANIFEST.JSON&version=GBmaster&line=152976&lineEnd=152976&lineStartColumn=13&lineEndColumn=24&lineStyle=plain&_a=contents).  Please add these new APIs into a new API version.

Reply:
the backend code is has already shipped as part of 2020-02-01 , we wanted to expose it in the latest api being released in Swagger so that our customers can start using the PS and AzCli cmdlets that we expose starting with this api version

Reply:
2021-03-01 has already been released.  Please add it to the next api version that isn't already available (or go through the breaking change review process in the PR description)